packages/shared/src/react/contexts.tsx

```diff
@@ -119,17 +120,10 @@ Learn more: https://clerk.com/docs/components/clerk-provider`.trim(),
 export {
   __experimental_CheckoutProvider,
   ClerkInstanceContext,
-  ClientContext,
   OptionsContext,
-  OrganizationProvider,
-  SessionContext,
   useAssertWrappedByClerkProvider,
   useCheckoutContext,
   useClerkInstanceContext,
-  useClientContext,
   useOptionsContext,
   useOrganizationContext,
-  UserContext,
```

Comment:
The contexts being removed is a breaking change, but we don't think anyone is using them, and if they do it's kind of unavoidable.